### ISSUE TITLE
`--json` flag for `unikube ps` command

### ISSUE THREAD

**Issue by SteinRobert** [unikubehq/cli#171]
For better "scriptability" it would be great to have a `--json` flag for the `unikube ps` command.
Otherwise users would have to parse the output of the command.

**Comment by schwobaseggl** [comment]
Are there commands for which that is the case? Should we offer json output in general?

**Comment by SteinRobert** [comment]
Yes I do this should be available for all commands. That would make the CLI way more scriptable, which is one of our goals for the cli in general.
